modules/kegg_completeness.nf

```diff
@@ -0,0 +1,42 @@
+process KEGG_COMPLETENESS {
+
+    label 'process_light'
+
+    publishDir(
+        path: "${params.outdir}",
+        saveAs: {
+            filename -> {
+                def cluster_prefix = cluster.substring(0, cluster.length() - 2);
+                return "species_catalogue/${cluster_prefix}/${cluster}/genome/${filename}";
+            }
+        },
+        mode: "copy",
+        failOnError: true
+    )
+
+    container 'https://depot.galaxyproject.org/singularity/kegg-pathways-completeness:1.0.5--pyhdfd78af_0'
```

Comment:
```suggestion
    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
        'https://depot.galaxyproject.org/singularity/kegg-pathways-completeness:1.0.5--pyhdfd78af_0':
        'biocontainers/kegg-pathways-completeness:1.0.5--pyhdfd78af_0' }"
```

Reply:
Done in https://github.com/EBI-Metagenomics/genomes-catalogue-pipeline/commit/942e8322318bdff30127482967369bb9604ab806